tests/cupy_tests/functional_tests/test_vectorize.py

```diff
@@ -477,6 +477,29 @@ def func_for(x):
         x = xp.array([0, 1, 2, 3, 4])
         return f(x)
 
+    @testing.numpy_cupy_array_equal()
+    def test_tuple(self, xp):
+        def func_tuple(x, y):
+            x, y = y, x
+            z = x, y
+            a, b = z
+            return a * a + b
+
+        f = xp.vectorize(func_tuple)
+        x = xp.array([0, 1, 2, 3, 4])
+        y = xp.array([5, 6, 7, 8, 9])
+        return f(x, y)
+
+    @testing.numpy_cupy_array_equal()
+    def test_return_tuple(self, xp):
```

Comment:
I suppose this test also implies the following operation will work fine as a `vectorize` target?
```python
>>> x = 5
>>> y = 3
>>> a, b = y-2, x-3
>>> print(a, b)
1 2
```

Reply:
Yes, the test of a tuple of expressions is covered in `test_return_tuple`.